### ISSUE TITLE
"Cannot resolve host name" but it's alive

### ISSUE THREAD

**Issue by worthmine** [tokuhirom/Furl#113]
<https://bank.teraren.com/> is an alive site, so we can visit it.

but with using Furl, 

```
$ perl -MFurl -e 'print Furl->new->get("https://bank.teraren.com/")->content'
Cannot resolve host name: bank.teraren.com (port: 443),  at -e line 1.
```

And I wonder why, getting JSON has succeeded like below:
 
```
$ perl -MFurl -e 'print Furl->new->get("https://bank.teraren.com/banks/0001.json")->content'
{"code":"0001","kana":"ミズホ","name":"みずほ","created_at":"2018-05-31T11:20:39.000Z","updated_at":"2018-05-31T11:20:39.000Z"}%
```
Is it the matter of DNS or this module or my settings?

- I DID upgrade cpan modules with `cpan-outdated | cpanm`
- the DNS record of this domain was updated on 06/21/2018
- I found this issue on 06/21/2018

If it was not caused by Furl, the first code I wrote above is a mystery.

**Comment by tokuhirom** [comment]
I can't reproduce your issue.

```
$ perl -MFurl -e 'print substr(Furl->new->get("https://bank.teraren.com/")->content, 0, 100)'
<!DOCTYPE html> <html lang="ja"> <head> <meta charset="utf-8"> <meta http-equiv="X-UA-Compatible" co
```

Maybe it caused by your environment? could you test it on another environment?

**Comment by worthmine** [comment]
Thank you for fast replying.

I tried twice to reproduce this issue after my post.
macOS 10.12.6 is my local environment, and I switched 5.18.4 to 5.22.3 and installed Furl and IO::Socket::SSL.
Then, no matter was found...

And I switched back to 5.18.4 and tried again.
Then, Actually, it worked with no error like you...

I'm sorry to be hurry. And agree with you, yes it's caused by my environment.

I will read this source well and find and fix my issues by myself. 

Thanks again.
